docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md

```diff
@@ -31,94 +46,194 @@ It applies to Intel Core Ultra and Core 12 - 14 gen integrated GPUs (iGPUs), as
 * Step 1: Open the **Anaconda Prompt** and activate the Python environment `llm` you previously created: 
    ```cmd
    conda activate llm
    ```
 * Step 2: Configure oneAPI variables by running the following command:
-  > For more details about runtime configurations, refer to [this guide](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#runtime-configuration):
   ```cmd
   call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat"
   ```
-  If you're running on iGPU, set additional environment variables by running the following commands:
-  ```cmd
-  set SYCL_CACHE_PERSISTENT=1
-  set BIGDL_LLM_XMX_DISABLED=1
+* Step 3:
+  Please also set the following environment variable according to your device:
+
+  ```eval_rst
+  .. tabs::
+     .. tab:: Intel iGPU
+  
+        .. code-block:: cmd
+  
+           set SYCL_CACHE_PERSISTENT=1
+           set BIGDL_LLM_XMX_DISABLED=1
+  
+     .. tab:: Intel Arc™ A300-Series or Pro A60
+  
+        .. code-block:: cmd
+  
+           set SYCL_CACHE_PERSISTENT=1
+  
+     .. tab:: Intel Arc™ A770 or Other Intel dGPU Series
+  
+        There is no need to set further environment variables.
+  ```
+  
+  ```eval_rst
+  .. seealso::
+
+    If you encountered any problem, please refer to `here <https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#troubleshooting>`_ for help.
   ```
-* Step 3: To ensure compatibility with `phi-1.5`, update the transformers library to version 4.37.0:
+* Step 4: Install additional package required for Qwen-1.8B-Chat to conduct:
    ```cmd
-   pip install -U transformers==4.37.0 
+   pip install tiktoken transformers_stream_generator
    ```
-* Step 4: Create a new file named `demo.py` and insert the code snippet below.
+* Step 5: Create a new file named `demo.py` and insert the code snippet below.
    ```python
    # Copy/Paste the contents to a new file demo.py
    import torch
    from bigdl.llm.transformers import AutoModelForCausalLM
    from transformers import AutoTokenizer, GenerationConfig
    generation_config = GenerationConfig(use_cache = True)
    
-   tokenizer = AutoTokenizer.from_pretrained("microsoft/phi-1_5", trust_remote_code=True)
+   print('Now start loading Tokenizer and Model...')
+   tokenizer = AutoTokenizer.from_pretrained("Qwen/Qwen-1_8B-Chat", trust_remote_code=True)
    # load Model using bigdl-llm and load it to GPU
    model = AutoModelForCausalLM.from_pretrained(
-       "microsoft/phi-1_5", load_in_4bit=True, cpu_embedding=True, trust_remote_code=True)
+       "Qwen/Qwen-1_8B-Chat", load_in_4bit=True, cpu_embedding=True, trust_remote_code=True)
    model = model.to('xpu')
+   print('Successfully loaded Tokenizer and Model!')
+
+   print('--------------------------------------Note----------------------------------------')
+   print('| When running LLMs on iGPU or Intel Arc™ A300-Series or Pro A60 for first time, |')
+   print('| it may take several minutes for GPU kernels to compile and initialize.         |')
```

Comment:
For the **first time** that **each model** runs on Intel iGPU/Intel Arc™ A300-Series or Pro A60, it may take several minutes for GPU kernels to compile and initialize.

Please be patient until it finishes warm-up...